### PR TITLE
Rke updates

### PR DESCRIPTION
**Pull request by deniseschannon**

### CODE REVIEW

File: content/rke/v0.1.x/en/config-options/_index.md [hunk 1]
```diff
@@ -4,49 +4,56 @@ weight: 3000
...
-`cluster_name` is an optional value that you can set to identify your cluster. The name will be set in your cluster's generated kubeconfig file.
-
-The default value for this option is `local`.
+When setting up your `cluster.yml` for RKE, there are a lot of different options that can be configured to control the behavior of how RKE launches Kubernetes.
+
+There are several options that can be configured in cluster configuration option. There are several [example yamls]({{< baseurl >}}/rke/v0.1.x/en/config-options/example-yamls/) that contain all the options.
+
+### Configuring Nodes
+* [Nodes]({{< baseurl >}}/rke/v0.1.x/en/config-options/nodes/)
+* [Ignoring supported Docker versions](#supported-docker-versions)
```
Comment (moelsayed) [inline content/rke/v0.1.x/en/config-options/_index.md]: unsupported*

File: content/rke/v0.1.x/en/config-options/nodes/_index.md [hunk 2]
```diff
@@ -34,43 +34,43 @@ nodes:
...
 
 The `internal_address` provides the ability to have nodes with multiple addresses set a specific address to use for inter-host communication on a private network. If the `internal_address` is not set, the `address` is used for inter-host communication.
 
-#### Overriding the Hostname
+### Overriding the Hostname
 
 The `hostname_override` is used to be able to provide a friendly name for RKE to use when registering the node in Kubernetes. This hostname doesn't need to be a routable address. If the `hostname_override` isn't set, then the `address` directive is used when registering the node in Kubernetes.
 
-> **Note:** When [cloud providers]({{< baseurl >}}/rke/v0.1.x/en/config-options/cloud-providers/) are configured, you may need to override the hostname in order to use the cloud provider correctly. 
+> **Note:** When [cloud providers]({{< baseurl >}}/rke/v0.1.x/en/config-options/cloud-providers/) are configured, you may need to override the hostname in order to use the cloud provider correctly.
```
Comment (moelsayed) [inline content/rke/v0.1.x/en/config-options/nodes/_index.md]: No, when cloud providers are used, the `hostname_override` is completely ignored. K8s get the hostname from the cloud provider itself.

File: content/rke/v0.1.x/en/config-options/services/extra-args-and-binds/_index.md [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+weight: 3026
 draft: true
 ---
 
+RKE supports the ability to configure your Kubernetes components by adding in extra service arguments to these components.
 
-RKE supports additional service arguments.
+## Extra Args
+
+For the `kube-controller`, you can update the `extra_args` to change the existing defaults.
```
Comment (moelsayed) [inline content/rke/v0.1.x/en/config-options/services/extra-args-and-binds/_index.md]: This looks like we are singling out kube-controller. All services support extra_args and extra_binds.

File: content/rke/v0.1.x/en/config-options/nodes/_index.md (hunk 2 above)
Comment (deniseschannon) [reply content/rke/v0.1.x/en/config-options/nodes/_index.md]: @galal-hussein i thought we discussed this way particularily for the azure cloud provider?

File: content/rke/v0.1.x/en/config-options/nodes/_index.md [hunk 4]
```diff
@@ -34,43 +34,43 @@ nodes:
...
 
 The `internal_address` provides the ability to have nodes with multiple addresses set a specific address to use for inter-host communication on a private network. If the `internal_address` is not set, the `address` is used for inter-host communication.
 
-#### Overriding the Hostname
+### Overriding the Hostname
 
 The `hostname_override` is used to be able to provide a friendly name for RKE to use when registering the node in Kubernetes. This hostname doesn't need to be a routable address. If the `hostname_override` isn't set, then the `address` directive is used when registering the node in Kubernetes.
 
-> **Note:** When [cloud providers]({{< baseurl >}}/rke/v0.1.x/en/config-options/cloud-providers/) are configured, you may need to override the hostname in order to use the cloud provider correctly. 
+> **Note:** When [cloud providers]({{< baseurl >}}/rke/v0.1.x/en/config-options/cloud-providers/) are configured, you may need to override the hostname in order to use the cloud provider correctly. There is an exception for the AWS cloud provider, where the `hostname_override` field will be explicitly ignored.
```
Comment (galal-hussein) [inline content/rke/v0.1.x/en/config-options/nodes/_index.md]: can you also add information of the node name that will be used in AWS, quoting from the documentation:
```
Concepts
Overview
What is Kubernetes?
Kubernetes Components
The Kubernetes API
Working with Kubernetes Objects
Understanding Kubernetes Objects
Names
Namespaces
Labels and Selectors
Annotations
Object Management Using kubectl
Kubernetes Object Management
Managing Kubernetes Objects Using Imperative Commands
Imperative Management of Kubernetes Objects Using Configuration Files
Declarative Management of Kubernetes Objects Using Configuration Files
Compute, Storage, and Networking Extensions
Cluster Administration Overview
Certificates
Cloud Providers
Managing Resources
Cluster Networking
Logging Architecture
Configuring kubelet Garbage Collection
Federation
Proxies in Kubernetes
Controller manager metrics
Installing Addons
Kubernetes Architecture
Nodes
Master-Node communication
Concepts Underlying the Cloud Controller Manager
Extending Kubernetes
Extending your Kubernetes Cluster
Extending the Kubernetes API
Extending the Kubernetes API with the aggregation layer
Custom Resources
Compute, Storage, and Networking Extensions
Network Plugins
Device Plugins
Service Catalog
Containers
Images
Container Environment Variables
Container Lifecycle Hooks
Workloads
Pods
Pod Overview
Pods
Pod Lifecycle
Init Containers
Pod Preset
Disruptions
Controllers
ReplicaSet
ReplicationController
Deployments
StatefulSets
DaemonSet
Garbage Collection
Jobs - Run to Completion
CronJob
Configuration
Configuration Best Practices
Managing Compute Resources for Containers
Assigning Pods to Nodes
Taints and Tolerations
Secrets
Organizing Cluster Access Using kubeconfig Files
Pod Priority and Preemption
Services, Load Balancing, and Networking
Services
DNS for Services and Pods
Connecting Applications with Services
Ingress
Network Policies
Adding entries to Pod /etc/hosts with HostAliases
Storage
Volumes
Persistent Volumes
Storage Classes
Dynamic Volume Provisioning
Policies
Resource Quotas
Pod Security Policies
Edit This Page

Cloud Providers
This page explains how to manage Kubernetes running on a specific cloud provider.

AWS
Azure
CloudStack
GCE
OpenStack
OVirt
Photon
VSphere
AWS
This section describes all the possible configurations which can be used when running Kubernetes on Amazon Web Services.

Node Name
The AWS cloud provider uses the private DNS name of the AWS instance as the name of the Kubernetes Node object.
```

**Pull request merged**